RabbitMq/AMQPConnectionFactory.php

```diff
@@ -30,7 +29,7 @@ class AMQPConnectionFactory
     /**
      * Constructor
      *
-     * @param string                                $class              FQCN of AMQPConnection class to instantiate.
+     * @param AbstractConnection                    $class              FQCN of AMQPConnection class to instantiate.
```

Comment:
This change require to pass object of class 'AbstractConnection' which is incorrect.
It should be 'class-string<AbstractConnection>'

See docs for class-string https://phpstan.org/writing-php-code/phpdoc-types#class-string

Reply:
Fixed. Thank you.